static/js/components/dashboard/courses/StatusMessages_test.js

```diff
@@ -393,7 +395,7 @@ describe("Course Status Messages", () => {
 
         const messages = calculateMessages(calculateMessagesProps).value
         assert.equal(messages[0]["message"], "You passed this course.")
-        assert.equal(messages[1]["message"], "You passed the exam.")
+        assert.equal(messages[1]["message"], "You passed the exam. ")
```

Comment:
If you follow my above comment, these space updates in the tests will not be needed.